examples/lock-app/nrfconnect/README.md

```diff
@@ -1,293 +1,336 @@
-    unprovisioned state and a commissioning application is connected via BLE.
+- _Short Flash On (50 ms on/950 ms off)_ &mdash; The device is in the unprovisioned (unpaired) state
+  and is waiting for a commissioning application to connect.
 
--   _Short Flash Off (950ms on/50ms off)_ &mdash; The device is fully
-    provisioned, but does not yet have full network (Thread) or service
-    connectivity.
+- _Rapid Even Flashing (100 ms on/100 ms off)_ &mdash; The device is in the unprovisioned state
+  and a commissioning application is connected through Bluetooth LE.
 
-*   _Solid On_ &mdash; The device is fully provisioned and has full network and
-    service connectivity.
+- _Short Flash Off (950ms on/50ms off)_ &mdash; The device is fully provisioned, but does not yet
+  have full Thread network or service connectivity.
 
-**Button #1** can be used to initiate a OTA software update as well as to reset
-the device to a default state.
+- _Solid On_ &mdash; The device is fully provisioned and has full Thread
+  network and service connectivity.
 
-Pressing and holding Button #1 for 6 seconds initiates a factory reset. After an
-initial period of 3 seconds, all four LED will flash in unison to signal the
-pending reset. Holding the button past 6 seconds will cause the device to reset
-its persistent configuration and initiate a reboot. The reset action can be
-cancelled by releasing the button at any point before the 6 second limit.
+**LED 2** simulates the lock bolt and shows the state of the lock.
+The following states are possible:
 
-**LED #2** shows the state of the simulated lock bolt. When the LED is lit the
-bolt is extended (i.e. door locked); when not lit, the bolt is retracted (door
-unlocked). The LED will flash whenever the simulated bolt is in motion from one
-position to another.
+- _Solid On_ &mdash; The bolt is extended and the door is locked.
 
-**Button #2** can be used to change the state of the simulated bolt. This can be
-used to mimick a user manually operating the lock. The button behaves as a
-toggle, swapping the state every time it is pressed.
+- _Off_ &mdash; The bolt is retracted and the door is unlocked.
 
-**Button #3** can be used to start Thread networking using default configuration
-which was selected to match OpenThread Border Router default settings and
-network credentials.
+- _Rapid Even Flashing (100 ms on/100 ms off during 2 s)_ &mdash; The simulated bolt is in motion
+  from one position to another.
 
-**Button #4** can be used to start BLE advertisement, which is disabled by
-default.
+**Button 1** can be used for the following purposes:
 
-The remaining two LEDs (#3 and #4) are unused.
+- _Pressed for 6 s_ &mdash; Initiates the factory reset of the device.
+  Releasing the button within the 6-second window cancels the factory reset procedure.
+  **LEDs 1-4** blink in unison when the factory reset procedure is initiated.
 
-**NFC** can be used to scan shared tag and get information about device
```

Comment:
Don't you think NFC antenna is also the UI, and should be listed together with Buttons, Leds and J-Link port?